frontend/src/metabase/query_builder/selectors.js

```diff
@@ -564,9 +564,7 @@ export const getRawSeries = createSelector(
     let display = question && question.display();
     let settings = question && question.settings();
 
-    if (isObjectDetail) {
-      display = "object";
```

Comment:
I'm a little concerned that there could be downstream side-effects of this, but I'm not sure where to look for them.